### PR TITLE
Netstandard

### PR DESCRIPTION
**Pull request by Jeern**
Hi,

I have a ASP.NET Core project that uses Markdig(which have a .NET Standard edition) , since your Nuget package Markdig.SyntaxHighlighting did not have a .NET Standard edition I could not use it,

In this PR:

I have changed the library from the old fashioned Portable library, to a .NET Standard library. Which can be used almost everywhere.

I have changed the Unittest project to a .NET Core project. None of your code has changed except one line in a unittest.

All tests run fine.

I have referenced the Markdig.SyntaxHighlighting.dll from my package and it also works fine there.

I have taken care to use your SharedAssemblyInfo.cs in the new  projects

In other words it should be Ok to accept this Pull Request, unless you still want to maintain the obsolete Portable library ?

Hope you can use it.

Regards Jesper

### CODE REVIEW

**Comment by Jeern** [comment]
I can see you have an AppVeyour build that would fail with my PR. Probably AppVeyour just needs to be configured to build it  in another way because it is a new project type.  Anyway - I still hope you can use the PR. It would certainly improve the reach of your Nuget package.
